runners/genlayer-py-std/src/genlayer/gl/genvm_contracts.py

```
@@ -51,7 +51,7 @@ def _make_calldata_obj(method, args, kwargs) -> calldata.Encodable:
 	return ret
 
 
-from ._internal.result_codes import StorageType
+from genlayer.py.public_abi import StorageType
```

Ruff (0.11.9)

Module level import not at top of file

(E402)